components/eamxx/src/share/property_checks/mass_and_energy_column_conservation_check.cpp

```diff
@@ -362,4 +528,17 @@ compute_energy_boundary_flux_on_column (const Real vapor_flux,
   return vapor_flux*(LatVap+LatIce) - (water_flux-ice_flux)*RHO_H2O*LatIce + heat_flux;
 }
 
+Real MassAndEnergyColumnConservationCheck::get_echeck(){
+  return echeck;
+}
+
+Real MassAndEnergyColumnConservationCheck::get_total_energy_before(){
+  return total_energy_before;
+}
+
+Real MassAndEnergyColumnConservationCheck::get_pb_fixer(){
```

Comment:
As Andrew said, we should prob find a different word than pb (I'm assuming it's "peanut butter"?). Maybe something more "mathematical", like "uniform fixer"? "mean value fixer"?